chrome/content/zotero/itemTreeColumns.jsx

```diff
@@ -23,65 +23,68 @@
- * 	staticWidth: boolean			// Default: false. Set to true to prevent columns from changing width when
- * 									// the width of the tree increases or decreases
- * 	minWidth: number,				// Override the default [20px] column min-width for resizing
- *
- * 	label: string,					// The column label. Either a string or the id to an i18n string.
- * 	iconLabel: React.Component,		// Set an Icon label instead of a text-based one
- *
- * 	ignoreInColumnPicker: boolean	// Default: false. Set to true to not display in column picker.
- * 	submenu: boolean,				// Default: false. Set to true to display the column in "More Columns" submenu of column picker.
- *
- * 	primary: boolean,				// Should only be one column at the time. Title is the primary column
- * 	zoteroPersist: Set<string>,		// Which column properties should be persisted between zotero close
- * 	}
+ * @typedef ItemTreeColumnOptions
+ * @type {object}
+ * @property {string} dataKey - Required, see use in ItemTree#_getRowData()
+ * @property {string[]} [enabledTreeIDs=[]] - Which tree ids the column should be enabled in. If undefined, enabled in every tree.
+ * @property {string[]} [defaultIn] - Will be deprecated. Types of trees the column is default in. Can be [default, feed];
+ * @property {string[]} [disabledIn] - Will be deprecated. Types of trees where the column is not available
+ * @property {number} [defaultSort=1] - Default: 1. -1 for descending sort
+ * @property {number} [flex=1] - Default: 1. When the column is added to the tree how much space it should occupy as a flex ratio
+ * @property {string} [width] - A column width instead of flex ratio. See above.
+ * @property {boolean} [fixedWidth] - Default: false. Set to true to disable column resizing
+ * @property {boolean} [staticWidth] - Default: false. Set to true to prevent columns from changing width when the width of the tree increases or decreases
+ * @property {number} [minWidth] - Override the default [20px] column min-width for resizing
+ * @property {string} label - The column label. Either a string or the id to an i18n string.
+ * @property {React.Component} [iconLabel] - Set an Icon label instead of a text-based one
+ * @property {string} [iconPath] - Set an Icon path, overrides {iconLabel}
+ * @property {boolean} [ignoreInColumnPicker=false] - Default: false. Set to true to not display in column picker.
+ * @property {boolean} [submenu=false] - Default: false. Set to true to display the column in "More Columns" submenu of column picker.
+ * @property {boolean} [primary] - Should only be one column at the time. Title is the primary column
+ * @property {boolean} [custom] - Set automatically to true when the column is added by the user
+ * @property {string} [pluginID] - Set plugin ID to auto remove column when plugin is removed
+ * @property {(item: Zotero.Item, dataKey: string, unformatted?: boolean) => string} [dataProvider] - Custom data provider that is called when rendering cells
```

Comment:
I'm still unclear why a plugin would need `unformatted`. A custom column has to be registered by the plugin, right? So won't it just be a custom `dataKey`, and isn't that sufficient?